### DISCUSSION TITLE
Can someone explain in simple terms what are the differences between Themes and Flavors and how they work together?

### DISCUSSION THREAD

**Discussion by onjour** [sxyazi/yazi#1035] [1. Q&A] [answered]
Hi all,

Basically the title. 

I'm very confused on how Themes and Flavors work and so far I only managed to change my theme by replacing my `*/.config/yazi/theme.toml`. 

Can someone give me an explanation and an example on how flavors work? I added a... 

```
[flavor]
use = "whatever"
```

Whereas the `*/.config/yazi/flavors/whatever.yazi` exists

...in my `*/.config/yazi/theme.toml` but I cannot see any difference. 

Thank you

**Comment by SolitudeSF** [comment]
you use theme to override parts of the flavor, so you dont need edit the flavor, if you only need a few values changed.

**Comment by sxyazi** [comment, marked as answer]
If you don't want to create your own color scheme and want to use someone else's, use flavor; otherwise, theme.

**Comment by onjour** [comment]
Thank you for your answers @sxyazi and @SolitudeSF. That helps!
